tar/chunks/chunks_stats.c

```diff
@@ -414,3 +414,45 @@ chunks_stats_free(CHUNKS_S * C)
 	free(C->cachepath);
 	free(C);
 }
+
+/**
+ * chunks_initialize(const char * cachepath):
+ * Initialize the chunk directory (file) in the ${cachepath}.
+ */
+int
+chunks_initialize(const char * cachepath)
+{
+	struct chunks_stats_internal * C;
+	void * dir;
+
+	/* Allocate memory. */
+	if ((C = malloc(sizeof(struct chunks_stats_internal))) == NULL)
+		goto err0;
+
+	/* Create a copy of the path. */
+	if ((C->cachepath = strdup(cachepath)) == NULL)
+		goto err1;
+
```

Comment:
Check that the ${cachepath}/directory doesn't exist.  We should throw an error if it does, because the user is probably making a mistake...